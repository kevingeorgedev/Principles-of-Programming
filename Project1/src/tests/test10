b=*/a24;!a;c=+5b;!b.
