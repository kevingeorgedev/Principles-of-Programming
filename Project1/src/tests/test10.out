loadI 1024 => r0
loadAI r0, 0 => r1
loadI 2 => r2
div r1, r2 => r3
loadI 4 => r4
mult r3, r4 => r5
storeAI r5 => r0, 4
outputAI r0, 0
loadI 5 => r6
loadAI r0, 4 => r7
add r6, r7 => r8
storeAI r8 => r0, 8
outputAI r0, 4
